workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/container/LayeredDialogSessions.kt

```diff
@@ -298,16 +301,36 @@ public class LayeredDialogSessions private constructor(
           "Expected a ViewTreeLifecycleOwner on $view"
         }
       }.also { dialogs ->
+        fun closeAll() {
+          dialogs.update(emptyList(), ViewEnvironment.EMPTY) {}
```

Comment:
an API that requires a lambda and yes we don't want to do anything in that lambda?

maybe that API needs tweaking? I have no idea what this is though.

Reply:
Non-normal call. I'm telling the `DialogManager` to show nothing, meaning close all Dialogs. The empty lambda is normally the one that updates the view underneat the dialogs. Nothing to update, we're closing up shop.